lib/serializer/JsonUpdater.cpp

```diff
@@ -67,7 +67,7 @@ void JsonUpdater::serializeInternal(const std::string & fieldName, double & valu
 		value = data.Float();
 }
 
-void JsonUpdater::serializeInternal(const std::string & fieldName, si64 & value, const boost::optional<si64> &)
+void JsonUpdater::serializeInternal(const std::string & fieldName, si64 & value, const boost::optional<si64> & /*defaultValue*/)
```

Comment:
I would rather un-comment this.
"unused variable" warning is disabled anyway

Reply:
done